src/main/java/com/foodapp/restaurant/Restaurant.java

```diff
@@ -0,0 +1,33 @@
+package com.foodapp.restaurant;
+
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.List;
+
+@Data
+@Document(collection = "restaurants")
+public class Restaurant {
+    @Id
+    private String id;
+    private String name;
+    private String cuisine;
+    private String address;
+    private int rating;
+    private List<String> opening_hours;
+    private String url;
+    private List<Object> reviews;
+    private String business_status;
+
+    public Restaurant(String name, String cuisine, String address, int rating, List<String> opening_hours, String url, List<Object> reviews, String business_status) {
+        this.name = name;
+        this.cuisine = cuisine;
+        this.address = address;
+        this.rating = rating;
+        this.opening_hours = opening_hours;
+        this.url = url;
+        this.reviews = reviews;
+        this.business_status = business_status;
+    }
+}
```

Comment:
@SodaJyu This is good! In the frontend client I used the word `establishment`. If Im not mistaken, there are a few fields that are different in the backend from a restaurant and say for example a movie theater right? If thats so, its probably for the best that we keep restaurants a type of establishment. Alternatively, Is it possible at a later time, to refactor so that we have an `establishment` parent that restaurant extends from? 

What are your thoughts? 

**Outside of restaurants are not part of MVP, so this isn't pressing.**

Reply:
@Jaymontojo This is definitely something we can look into doing as we move beyond MVP! Should be an easy thing to implement!